tests/unit/Rules/JsonTest.php

```diff
@@ -11,28 +11,23 @@
 
 namespace Respect\Validation\Rules;
 
+use Respect\Validation\Test\RuleTestCase;
```

Comment:
Just remove this line, `RuleTestCase` is in the same namespace of the test (and the rule).